src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Webservice/WebserviceKeyType.php

```diff
@@ -111,6 +112,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                             'Admin.Advparameters.Notification'
                         ),
                     ]),
+                    new Regex([
+                        'pattern' => '/^[a-zA-Z0-9@\#\?\-\_]+$/i',
```

Comment:
Use a CONST for this pattern ?

Reply:
Yeah, why not, do you think it's better for future changes?
Otherwise I don't see a big difference 😊